app/src/main/java/com/example/giphyservice/data/GifService.kt

```diff
@@ -1,10 +1,10 @@
 package com.example.giphyservice.data
 
 import com.example.giphyservice.data.model.DataResult
-import retrofit2.Call
+import retrofit2.Response
 import retrofit2.http.GET
 
 interface GifService {
     @GET("gifs/trending?api_key=ROTqe5iNDVl5ti1dUCVXDF1E19rHrHXR")
-    fun getGifs(): Call<DataResult>
+    suspend fun getGifs(): Response<DataResult>
```

Comment:
you can avoid `Response`. Your code can failed with HttpException in case of HTTP error or return required data in case of success. So you can easily omit Response. and then handle exceptions in repository using try-catch/runCatching